File: lab/javascript/POO/pessoa2.js
class Pessoa{
    constructor(nome, idade){
        this.nome = nome;
        this.idade = idade;
    }

    info(){
        return `Nome: ${this.nome}, Idade: ${this.idade}`;
    }

    apresentar(){
        console.log(`Olá meu nome é ${self.nome}`);
    }
}

p = new Pessoa("João", 21)
console.log(p.info())
p.apresentar()

File: lab/python/POO/pessoa2.py
class Pessoa:
    def __init__(self, nome, idade) -> None:
        self.nome = nome
        self.idade = idade

    def __repr__(self) -> str:
        return f"Nome: {self.nome}, Idade: {self.idade}"

    def apresentar(self) -> None:
        print(f"Olá meu nome é {self.nome}")


p = Pessoa("João", 21)
print(p)
p.apresentar()
